docs/DeveloperGuide.adoc

```diff
@@ -337,10 +421,146 @@ Use case ends.
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. Mortago shows an error message.
 +
 Use case resumes at step 2.
 
+[discrete]
+=== Use case: Find entry
+*MSS*
+
+1.  Mortuary manager switches to the desired view (bodies or workers).
+2.  Mortuary manager specifies word to search.
+3.  Mortago shows a list of entries whose names matches the word.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 3a. The list is empty.
++
+Use case ends.
+
+[discrete]
+=== Use case: Filter entries
+*MSS*
+
+1.  Mortuary manage switches to the desired view (bodies or workers).
+2.  Mortuary manager specifies criteria for filter.
+3.  Mortago shows a list of entries that matches the criteria.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 3a. The list is empty.
++
+Use case ends.
+
+[discrete]
+=== Use case: Sort entries
+*MSS*
+
+1.  Mortuary manage switches to the desired view (bodies or workers).
+2.  Mortuary manager specifies criteria for sorting.
+3.  Mortago shows a list of entries sorted according to the specified criteria.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 3a. The list is empty.
```

Comment:
This extension already falls under the MSS since there is nothing to show. 
Maybe could change it to: Application displays error indicating that there are no bodies in the list to displauy